chat/chat.py

```diff
@@ -9,28 +9,55 @@
     -u <name>, --username <name>    The user nickname [default: guest].
     -h, --help                      Show this message and exit.
 """
-
```

Comment:
The exercise originaly built on TCP, it has more logic (client/server and syncrhonization), if you want to learn much more about asyncio concepts go ahead.